content/docs/quickstart/hasura.md

```diff
@@ -28,13 +48,19 @@ query MyQuery {
 }
 ```
 
-Save this GraphQL query as an HTTP API endpoint by clicking the REST tab. Let us call the endpoint 'query_t'.
+Then, save this GraphQL query as an HTTP API endpoint by clicking the REST tab. Let us call the endpoint 'query_t'.
+
+Finally, you can use this endpoint to get the table content now.
 
-Finally, you can use this endpoint to get the table content now. For example, that is how it works from the shell:
+Your output on the screen should be similar to the example below:
 
 ```bash
 $ curl -H 'x-hasura-admin-secret: {admin_secret}' https://{your_project_name}.hasura.app/api/rest/query_t
 {"t":[{"text":"test"}]}
 ```
 
 Thanks for your time, and keep hacking!
+
+## Conclusion
+
+In this tutorial, you performed an integration between Neon cloud service and Hasura Cloud. Now you can use an HTTP API endpoint that uses GraphQL to query Neon's serverless database and returns a set of rows.
```

Comment:
IMO, it looks a bit fake / lazy when Conclusion == Introduction in the past :) I guess that other people may feel the same.

I'd sprinkle some amazingness over here, like mention the serverless nature of the service again.

I.e. mention that Neon's Postgres isn't running all the time. And whenever (1 hour, 1 day, 1 month) you come back to your Hasura API our will be spun up again in seconds and ready to serve requests. This isn't a case with Planetscale for example, where you have to go to the control panel and start it manually after 1 month or so.

But that's just IMO. Another option is to just remove Conclusion at all

Reply:
@ololobus Thank you for the feedback!

As part of our documentation cleanup and site re-organization, we will not be writing new content unless this has changed @stepashka, and we can discuss what next content you would like written.

I will remove the Conclusion in this case.